cluster/images/hyperkube/Dockerfile

```diff
@@ -14,5 +14,10 @@
 
 FROM BASEIMAGE
 
-# Copy the hyperkube binary
+# Cleanup all the soft links
+ADD binaries.tgz /usr/local/bin
```

Comment:
is /usr/local/bin in the $PATH for this image?

Reply:
yes it is.